appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -839,7 +839,9 @@ private YaVersion() {
   // - The Alignment property was renamed to TextAlignment.
   // For PASSWORDTEXTBOX_COMPONENT_VERSION 3:
   // - Added RequestFocus Function (via TextBoxBase)
-  public static final int PASSWORDTEXTBOX_COMPONENT_VERSION = 3;
+  // For PASSWORDTEXTBOX_COMPONENT_VERSION 4:
+  // - Added PasswordVisible property (via TextBoxBase)
```

Comment:
According to this comment you added PasswordVisible via TextBoxBase, but that is not true according to the code. Would you mind changing the comment to reflect this?